packet/client.go

```diff
@@ -62,6 +73,7 @@ func NewClient(consumerToken, authToken string, baseURL *url.URL) (*Client, erro
 		if err != nil {
 			return nil, errors.Wrap(err, "connect to cacher")
 		}
+		wg = nil
```

Comment:
this assignment is ineffective, wg is already nil here anyway.

Reply:
done, removed

Reply:
done